pkg/sql/logictest/logic.go

```diff
@@ -3999,6 +4000,9 @@ type TestServerArgs struct {
 	// If set, then we will disable the metamorphic randomization of
 	// smallEngineBlocks variable.
 	DisableSmallEngineBlocks bool
+	// If set, then default-batch-bytes-limit metamorphic constant is guaranteed
+	// to be at least 1KiB.
+	EnableLowerBoundOnBatchBytesLimit bool
```

Comment:
[nit] could you make this the lower bound in bytes instead, and just set it to 1KiB here?

Reply:
Done. I also decided to bump it up to 3KiB min.